.github/workflows/ci.yml

```diff
@@ -1,6 +1,7 @@
 name: CI
 
 on:
+  workflow_dispatch:
```

Comment:
This allows you to manually run your GItHub CICD workflow from the Actions tab without pushing a commit. If you would prefer me to revert this change I can. Just LMK.